src/Components/SidebarItems/Sidebar.css

```diff
@@ -0,0 +1,35 @@
+@import "./../../style.css";
+.sidebar{
+    height: 37rem;
+    width: 14rem;
+    border: 1px solid var(--dark-black-color);
+    overflow: auto;
+    background-color: var(--light-black-color);
+    overflow-x: hidden;
+}
+.sidebarItemsLink{
+    display:flex ;
+    width:  14rem;
+    text-decoration: none;
+    padding: 1rem;
+    flex-wrap: wrap;
+    font-size:1rem;
+    color: var(--white-color);
+    font-weight: bolder;
+    margin-top: 0.5rem ;
+    margin-left: 1rem;
+}
+.sidebarItemsLink:hover{
+    background-color:var(--gray-text-color) ;
+}
+
+.material-icons,.fa,
+.fas{
```

Comment:
Can use different class names instead of the ones provided by the icons packages!!